Sources/Coordinator/CoordinatorUpdate/ContainerCoordinatorUpdate.swift

```diff
@@ -278,31 +280,31 @@ extension ContainerCoordinatorUpdate {
         var values = ContiguousArray<Value>(capacity: targetValues.count)
         var indices = Indices(capacity: self.sourceIndices.count)
         var result = Result(), change: (() -> Void)?
-        var offset: O { .init(context.offset?.offset.target, offset: indices.count) }
+        var target: O { .init(context.offset?.offset.target, offset: indices.count) }
```

Comment:
The identifier name `offset` is used for `let offset = offsets[toIdentifier(value)]` below and I cannot figure out a better name rather than `target` here. Not sure if `target` is a suitable one.